changelog.txt

```diff
@@ -1,3 +1,9 @@
+Version 1.30.0-beta
+=============
+- Replace org.projectlombok dependencies with implementations of generated code (#946)
```

Comment:
We should mention about breaking changes from exception changes

Reply:
Normally we just make a separate changelog/version update PR once everything is merged to clearly mark the cutoff for changes that we release, so I didn't include them here

But this is kind of a special case due to it already being released. I just set up this PR to make it easier to review the changelog, and will either merge into this branch or the main branch depending on which gets approved first: https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/983